inc/customizer/colors.php

```diff
@@ -1322,6 +1322,25 @@ public function get_color_overlay_transparency_value() {
 
 	}
 
+	/**
+	 * Magic getter for colors and color_schemes property
+	 *
+	 * @param $name
+	 *
+	 * @return bool
+	 */
+	public function __get( $name ) {
+
+		if ( ! in_array( $name, [ 'colors', 'color_schemes' ] ) ) {
```

Comment:
@EvanHerman I feel better with readonly property. They should use filters if they want to modify anything